app/src/main/res/layout/history_metadata_header.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="56dp"
+    android:layout_marginTop="40dp">
```

Comment:
Can we change this to 20dp in all three headers? There's currently a massive gap between all those sections and they are already disappearing below the fold.

Reply:
Will follow up on this separately after we speak to UX.

Reply:
We can bring this up separately and verify.